src/parser/rogue/outlaw/modules/spells/RollTheBonesBuffs.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+import Analyzer from 'parser/core/Analyzer';
+import UptimeIcon from 'interface/icons/Uptime';
+import StatisticBox from 'interface/others/StatisticBox';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+
+import { ROLL_THE_BONES_BUFFS } from '../../constants';
+
+class RollTheBonesBuffs extends Analyzer {
+  constructor(...args) {
+    super(...args);
+    this.active = !this.selectedCombatant.hasTalent(SPELLS.SLICE_AND_DICE_TALENT.id);
+  }
+
+  percentUptime(spellid) {
+    return this.selectedCombatant.getBuffUptime(spellid) / this.owner.fightDuration;
```

Comment:
Yep it should. Referencing my other comment on the cast tracker, all of the buff checking should still be accurate even without the refresh events for specific buffs. I didn't include overall buff uptime in the tracker since I didn't want to duplicate functionality that was already available